### ISSUE TITLE
Aktualizovat dokumentaci

### ISSUE THREAD

**Issue by paukert** [Quick-Event/quickbox#725]
- [ ] zdokumentovat, jak je možné upravit lísteček s mezičasy / diplomy
- [ ] lépe popsat nastavení PostgreSQL
- [ ] přidat do known issues problém s přidáváním závodníka (je do databáze vložen po kliknutí na tlačítko E1, tlačítko Zrušit závodníka z databáze neodstraní)
- [ ] popsat jak fungují losovací metody
- [ ] popsat práci s vakanty
- [ ] postupně projít všechny stránky a provést celkovou aktualizaci
- [ ] vytipovat stránky, které je možné odstranit (a po odsouhlasení wiki pročistit)

**Comment by paukert** [comment]
V rámci úklidu bych odstranil z Wiki tyto stránky, dává vám to také smysl?

https://github.com/Quick-Event/quickbox/wiki/ORIS-soupisky-%C5%A1tafet-export
https://github.com/Quick-Event/quickbox/wiki/Presentation-%C5%A0ejdorfsk%C3%BD-ml%C3%BDn
https://github.com/Quick-Event/quickbox/wiki/Release-plan
https://github.com/Quick-Event/quickbox/wiki/relays-courses-export-v8
https://github.com/Quick-Event/quickbox/wiki/%C5%A0tafety-v%C3%BDvoj

**Comment by otahirs** [comment]
za mě maž a upravuj dle sveho nejlepsiho vedomi a svedomi :)

**Comment by paukert** [comment]
Jo, je tam hodně práce na úpravu (a vytvoření nových návodů). Chtěl bych to postupně začít všechno procházet a trošku upgradnout 😄 

@fvacek nevíš, jestli bys mi nemohl nastavit právo na aktualizaci Wiki přes git? Repozitář bez problému naklonuju (`git clone git@github.com:Quick-Event/quickbox.wiki.git quickbox-wiki`), ale na push nemám právo. Bylo by to trošku pohodlnější, mohl bych upravovat více stránek najednou a dávat screeny z programu přímo do wiki repozitáře (a nemusely by být někde stranou).

**Comment by fvacek** [comment]
> Jo, je tam hodně práce na úpravu (a vytvoření nových návodů). Chtěl bych to postupně začít všechno procházet a trošku upgradnout smile
> 
> @fvacek nevíš, jestli bys mi nemohl nastavit právo na aktualizaci Wiki přes git? Repozitář bez problému naklonuju (`git clone git@github.com:Quick-Event/quickbox.wiki.git quickbox-wiki`), ale na push nemám právo. Bylo by to trošku pohodlnější, mohl bych upravovat více stránek najednou a dávat screeny z programu přímo do wiki repozitáře (a nemusely by být někde stranou).

Moc rad, jenom nevim, jestli se mi to podari na prvni pokus :)

**Comment by fvacek** [comment]
@paukert  zkus to ted

**Comment by paukert** [comment]
> @paukert zkus to ted

Funguje, díky :)
